tests/scenarios/test_functionality_nimbus.py

```diff
@@ -126,3 +137,90 @@ def check_new_tab(selenium):
     # get the last tab
     selenium.switch_to.window(selenium.window_handles[-1])
     assert "about:newtab" in selenium.current_url
+
+
+@then("The user will install a language pack")
+def install_acholi_language_pack(selenium, request):
+
+    add_button_locator = (By.CSS_SELECTOR, "#add")
+    addon_installed_locator = (By.CSS_SELECTOR, "#appMenu-addon-installed-notification")
+    add_to_firefox_locator = (
+        By.CSS_SELECTOR,
+        "#addon-install-confirmation-notification .popup-notification-primary-button",
+    )
+    browser_dialog_box_locator = (By.CSS_SELECTOR, "#BrowserLanguagesDialog")
+    root_dialog_box_locator = (
+        By.CSS_SELECTOR,
+        "vbox.dialogOverlay:nth-child(1) > vbox:nth-child(1) > browser:nth-child(2)",
+    )
+    language_button_locator = (By.CSS_SELECTOR, "#manageBrowserLanguagesButton")
+    language_search_locator = (By.CSS_SELECTOR, ".in-menulist menuitem label")
+    menu_list_locator = (By.CSS_SELECTOR, ".languages-grid #availableLocales .in-menulist")
+
+    if not request.config.getoption("--run-firefox-release"):
+        pytest.skip("needs --run-firefox-release option to run")
```

Comment:
😍 

Actually&hellip; should this be higher up? Why `.skip()` here and not above at the beginning of the method on L144? Why bother with all the selectors (L145-158) only to bail here if you aren't in Release channel?

Reply:
The selectors aren't technically doing anything, I dont think it would make much of a difference